doc/devel/coding_style_guideline.rst

```diff
@@ -118,22 +118,22 @@ body::
 
 Using C libraries
 -----------------
-The `cimport`s should follow the same rules defined in PEP8 for `import`
-statements. If a module is both *imported* and *cimported*, the `cimport`
-should come before the `import`.
+The ``cimport`s`` should follow the same rules defined in PEP8 for ``import``
```

Comment:
@areeshatariq Sorry again, I failed to spot this one: the first one should be (double_backticks)cimport(double_backticks)s

Reply:
Same as before: my bad. So this will work: (double_backticks)cimport(double_backticks)'s (note the straight tick before the s).